priv/repo/migrations/20200710182224_create_results.exs

```diff
@@ -0,0 +1,20 @@
+defmodule LiquidVoting.Repo.Migrations.CreateResults do
+  use Ecto.Migration
+
+  def change do
+    create table(:results) do
+      add :in_favor, :integer, default: 0
+      add :against, :integer, default: 0
+      add :proposal_url, :string, default: false, null: false
+      add :organization_uuid, :uuid
```

Comment:
I'm wondering if we should change this to `organisation_id` already (and everywhere else)?

Reply:
I created a new issue for this, yesterday.

Reply:
This is now done, and related new issue closed.